packages/round-manager/src/features/api/utils.ts

```diff
@@ -3,8 +3,90 @@ import { ApplicationMetadata, InputType, IPFSObject } from "./types";
 export enum ChainId {
   GOERLI_CHAIN_ID = 5,
   OPTIMISM_MAINNET_CHAIN_ID = 10,
+  FANTOM_MAINNET_CHAIN_ID = 250,
 }
 
+export type PayoutToken = {
+  name: string;
+  chainId: number;
+  address: string;
+  logo?: string;
+  default?: boolean; // TODO: this is only used to provide the initial placeholder item, look for better solution
+};
+
+export const TokenNamesAndLogos: Record<string, string> = {
+  FTM: "https://picsum.photos/256",
+  BUSD: "https://picsum.photos/256",
+  DAI: "https://picsum.photos/256",
+  ETH: "https://picsum.photos/256",
+};
+
+export const getPayoutTokenOptions = (chainId: ChainId): PayoutToken[] => {
+  switch (chainId) {
+    case ChainId.OPTIMISM_MAINNET_CHAIN_ID: {
+      return [
+        {
+          name: "DAI",
+          chainId: ChainId.OPTIMISM_MAINNET_CHAIN_ID,
+          address: "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+          logo: TokenNamesAndLogos["DAI"],
+        },
+        {
+          name: "ETH",
+          chainId: ChainId.OPTIMISM_MAINNET_CHAIN_ID,
+          address: "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000",
```

Comment:
is this the correct token address? was referencing:

https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/5ddde7360f7bebfbeeab55abba069c4ca27d033e/optimism.tokenlist.json#L814-L824